### ISSUE TITLE
125mm inlet/outlet

### ISSUE THREAD

**Issue by phillipsnick** [Makerr-Studio/CentrifugalDustSeparator-#2]
Hey, thanks for your work on this! My printer is busy printing all the internal parts and tube/o rings have been shipped.

I've tried to make the changes in Fusion 360 but it's way above my skill level. Are there any plans in the future for Y splitters with 125mm inlet/outlet?

**Comment by asucks** [comment]
hey Phil,

Others can chime in, but I would hold off on changing the sizes on anything for now.  I have a much larger inlet for my dust collector as well, but what I've learned over the years is bigger is not always better.  

Your dust extractor might have a 125mm inlet, but it might not be able to maintain the pressure/speeds required to have the centrifuge run efficiently at 125mm, as it could be magnitudes difference in the volume of air. 

That's why @Makerr-Studio is talking about his dream system being some sort of hybrid, one that can handle a high volume of air (high CFM with big inlets), but also one that maintains high speeds and pressure like those with smaller inlets.

Check out this video, this dude (and his marbles) can explain it better than I can: 

https://www.youtube.com/watch?v=0NmMoIYIsr8

**Comment by Makerr-Studio** [comment]
Hey Phil,

I have the CAD-files available on my computer! I will add those to the files soon. I tested my collector over 100mm tubing. I put the 100mm hose inside the 125mm Y-splitter. So I do not have good test results on that. 

I need to order the tubing for that. Now I reduce at the planer from 125mm to 100mm. My goal is a 125mm inlet system attached to the 100mm camvac. Maybe the separation of the dust is enough to compensate for the volume change. Still need to test that.

**Comment by Makerr-Studio** [comment]
I added the files to the following folder: MK2/NOT TESTED - REQUESTED PARTS

**Comment by phillipsnick** [comment]
hey @asucks 

Really ELI5 video that, thanks for sharing!

I'm still learning a lot about all this, but given than a smaller duct means more friction, wouldn't it make sense to keep things consistent from the fan where possible? 
My extractor is rated at 2HP with a flowrate of 2000m3/hour (1177 CFM). So does have some power behind it!

**Comment by phillipsnick** [comment]
@Makerr-Studio thanks for putting them together, will get printing over the next week.

If I have some left over filament I'll try print the originals too and do a side by side comparison and share some results if I have time!
